Ports/quake3/package.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=quake3
+version=1.34
+useconfigure=true
+commit_hash=6d74896557d8c193a9f19bc6845a47e9d0f77db2
+archive_hash=1db91cfd05170ed5b37c1ab56cdf7bbe6b3c86fc6baee8b68e8e539fddfd88c1
+files="https://github.com/ioquake/ioq3/archive/$commit_hash.tar.gz ioq3.tar.gz $archive_hash"
+auth_type=sha256
+workdir="ioq3-${commit_hash}"
+depends=("SDL2")
+launcher_name="QuakeIII: Arena"
+launcher_category="Games"
+launcher_command=/usr/local/games/quake3/ioquake3
+icon_file="misc/quake3.png"
+
+configure() {
```

Comment:
This is the default `build` function isn't it?

Reply:
So it is. I'll remove that.